log/init.lua

```diff
@@ -14,43 +14,44 @@ local doc = require 'argcheck.doc'
 local Log = torch.class('tnt.Log', tnt)
 
 doc[[
-### Log
+### tnt.Log
 
-Log classes act as tables indexed by string keys. Allowed keys must be
+`Log` classes act as tables indexed by string keys. Allowed keys must be
 provided at construction. A special key `__status__` can be also set the
 convenience method `log:status()` to record basic messages.
 
-Viewers closures can be attached to a Log, and called at different events:
-   * `onSet(log, key, value)`: when setting a key to the Log with `log:set{}`.
+Viewers closures can be attached to a `Log`, and called at different events:
+   * `onSet(log, key, value)`: when setting a key to the `Log` with `log:set{}`.
    * `onGet(log, key)`: when querying a key with `log:get()`.
-   * `onFlush(log)`: when flushing out the stored data of the Log with `log:flush()`.
-   * `onClose(log)`: when closing a Log with `log:close()`.
+   * `onFlush(log)`: when flushing out the stored data of the `Log` with `log:flush()`.
+   * `onClose(log)`: when closing a `Log` with `log:close()`.
 
 Typical viewer closures are `text` or `json`, which allow to write to disk
-or to the console a subset of the keys stored by the Log, in a particular
+or to the console a subset of the keys stored by the `Log`, in a particular
 format. The special viewer closure `status` is made to be called on `set()`
 events, and will print out only status records.
 
 A typical use case would be the following:
 ```lua
+tnt = require 'torchnet'
 
 -- require the viewers we want
-local logtext = require 'torchnet.log.view.text'
-local logstatus = require 'torchnet.log.view.status'
+logtext = require 'torchnet.log.view.text'
```

Comment:
It won't run in the REPL... These are examples people copy & paste into the terminal.